x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -534,3 +534,27 @@ process where subtract(43, serial_event_id) == 41
 InternalQlScriptUtils.sub(params.v0,InternalQlScriptUtils.docValue(doc,params.v1)),params.v2))",
 "params":{"v0":43,"v1":"serial_event_id","v2":41}
 ;
+
+eventQueryDefaultLimit
```

Comment:
@jrodewig tail and pipe should now work for all type of queries; if you encountered problems when trying to use them in the docs let me know since that would be a bug.